data/mods/DinoMod/monsters/juvenile.json

```diff
@@ -297,9 +337,10 @@
   {
     "id": "mon_brachiosaurus_juvenile",
     "type": "MONSTER",
-    "name": "brown and magenta four-legged juvenile",
+    "name": "brown and magenta sauropod juvenile",
     "description": "A huge four-legged plant-eating juvenile sauropod with a long neck and tail, it could be from a number of different species.",
```

Comment:
```suggestion
    "description": "A huge, four-legged, plant-eating juvenile sauropod with a long neck and tail, it could be from a number of different species.",
```